test/pylib/manager_client.py

```diff
@@ -102,3 +102,8 @@ async def servers(self) -> List[str]:
         """Get list of running servers"""
         host_list = await self._request("http://localhost/cluster/servers")
         return host_list.split(',')
+
+    async def mark_dirty(self) -> None:
+        """Manually mark current cluster dirty.
+           To be used when a server was modified outside of this API."""
+        await self._request("http://localhost/cluster/mark-dirty", "Could not mark cluster dirty")
```

Comment:
@kostja explained to me in the sister PR that the hostname in the request is just ignored (and probably gets sent as a "Host:" header if it uses the real HTTP protocol) - the requests get sent over the unix domain socket. So it can be "localhost" or anything at all like "unixsocket" or "this.doesnt.mean.anything".

I wonder if this can't be cleaned up by "_request" (why the underscore, by the way?) not taking a URL and rather just taking a "path" (the /cluster/mark-dirty" part). This function can add the silly "http://localhost" to every path it gets. I think It will make the code a bit less misleading for readers.

Reply:
Sounds good. I'll do it in the next pull.

Reply:
I found there's an unofficial convention on Unix domain sockets, at least in some `Python` packages like `requests`:
`http+unix://%2Ftmp%2Fprofilesvc.sock/`. It seems to work with this as the socket file path works as hostname.

Reply:
But since the socket file path is constant in all our code (comes from some global variable), I think it makes sense not to have to repeat it in every request.